job-tutorial/README.md

```diff
@@ -106,7 +106,7 @@ Below please find the detailed explanation for each of the parameters in the con
 | `authFile`                     | String, optional, HDFS URI | Docker registry authentication file existing on HDFS |
 | `dataDir`                      | String, optional, HDFS URI | Data directory existing on HDFS          |
 | `outputDir`                    | String, optional, HDFS URI | Output directory on HDFS, `$PAI_DEFAULT_FS_URI/Output/$jobName` will be used if not specified |
-| `codeDir`                      | String, optional, HDFS URI | Code directory existing on HDFS          |
+| `codeDir`                      | String, optional, HDFS URI | Code directory existing on HDFS, should not contain any data and should be less than 10MB     |
```

Comment:
>10MB  [](start = 149, length = 5)

Is 10M is too small? I suggest 1G would be a more user friendly size. In future, this number should be adjustable from configuration, suggest to add a TODO here to remind this thing.

Reply:
10M may be not enough? Jobs may include some demo images or other binary files for preprocess

Reply:
I suggest at least set it to 100M~200M. Remember we not only support DL workloads. Maybe some program has large footprint (like resource files)

---
In reply to: [205726533](https://github.com/Microsoft/pai/pull/975#discussion_r205726533) [](ancestors = 205726533)